src/D2L.CodeStyle.Analyzers/Async/Generator/SyntaxTransformer.cs

```diff
@@ -97,15 +97,28 @@ ImmutableArray<Diagnostic> Diagnostics
 	/// </summary>
 	protected SyntaxTokenList TransformAll(
 		SyntaxTokenList input,
-		Func<SyntaxToken, SyntaxToken> transformer

```

Comment:
Eh I screwed up with this stuff. Went back to having a separate impl for `SyntaxToken?` and using `null` to single filtering out the token. Copy and paste but whatever.